src/compute/src/render.rs

```diff
@@ -343,10 +344,16 @@ pub fn build_compute_dataflow<A: Allocate>(
                             let depends = object.plan.depends();
                             context
                                 .enter_region(region, Some(&depends))
-                                .render_recursive_plan(0, object.plan)
+                                .render_recursive_plan(object.id, 0, object.plan)
                                 .leave_region()
                         },
                     );
+                    let global_id = object.id;
+                    if let Some(id) = bundle.scope().addr().first() {
+                        context.log_dataflow_global_id(*id, global_id);
+                    } else {
+                        tracing::error!(%global_id, "could not find dataflow, bundle address was empty");
+                    }
```

Comment:
If this would happen, we'd crash for sure, so you can just expect it:
```suggestion
                    context.log_dataflow_global_id(*bundle.scope().addr().first().expect("Must exist"), global_id);
```